cli/exchange/node.go

```diff
@@ -618,6 +620,33 @@ func NodeListStatus(org string, credToUse string, node string) {
 
 }
 
+// NodeServiceConfigStateList list of service config state of a node.
+func NodeServiceConfigStateList(org string, credToUse string, node string) {
+	nodeOrg, node := cliutils.TrimOrg(org, node)
+	var nodes ExchangeNodes
+	httpCode := cliutils.ExchangeGet("Exchange", cliutils.GetExchangeUrl(), "orgs/"+nodeOrg+"/nodes"+cliutils.AddSlash(node), cliutils.OrgAndCreds(org, credToUse), []int{200, 404}, &nodes)
+	if httpCode == 404 && node != "" {
+		cliutils.Fatal(cliutils.NOT_FOUND, i18n.GetMessagePrinter().Sprintf("node '%s' not found in org %s", node, nodeOrg))
+	}
+	nodeKey := fmt.Sprintf("%s/%s", org, node)
+	output := cliutils.MarshalIndent(nodes.Nodes[nodeKey].RegisteredServices, "exchange node list")
```

Comment:
This does not produce the same output as hzn service configstate list command. You will need to take the output of the exchange API call and transform it into the same output object as used by hzn service configstate list.